helpers/utils.h

```diff
@@ -346,6 +346,8 @@ namespace Utils
 
     bool isPluginActivated(const char* callSign);
 
+    bool isPluginUnavailable(const char* callSign);
```

Comment:
Interface (method definitions) are so important for developpers who consume it. 
This definition to me implies 2 things. 
1) If it is a static method, feels like legacy C code, there you expect static methods. Also what would happen if I feed it a nullptr, whats the use case for that. Since it does not say that it is expecteing an array of characters but a pointer to characters, I assume the nullptr has a meaning, but what ?
2) If it is part of an object, it feels as C++ code. But than why is this function not const? Is there instance object changes if I call this method? Or did they just forget the const at the end of the method. Would the signature actually be: bool IsPluginUnavailable(const TCHAR callsign[]) const; ?